src/OpenTelemetry.Instrumentation.StackExchangeRedis/OpenTelemetry.Instrumentation.StackExchangeRedis.csproj

```diff
@@ -17,7 +17,6 @@
   <ItemGroup>
     <PackageReference Include="OpenTelemetry.Api" Version="$(OpenTelemetryApiPkgVer)" />
     <PackageReference Include="StackExchange.Redis" Version="$(StackExchangeRedisPkgVer)" />
-    <PackageReference Include="Microsoft.Extensions.Options" Version="$(MicrosoftExtensionsOptionsPkgVer)" />
```

Comment:
@ArminShoeibi, it will not work in this way.
The shared package is not references by this project. File `ServiceProviderExtensions` is compiled directly into this project. You cannot remove this reference without removing ServiceProviderExtensions. - please check line 13.

Reply:
Oh, I got it.
That is a nice way of referencing without ProjectReference,
Thanks